padd.sh

```diff
@@ -15,7 +15,7 @@ export LC_NUMERIC=C
 ############################################ VARIABLES #############################################
 
 # VERSION
-padd_version="v3.10.1"
+padd_version="v3.1.1"
```

Comment:
Please, do not change the version number.

Reply:
Must've been from testing, didn't mean to push it.